src/model/response/BookAvailabilityResponse.java

```diff
@@ -0,0 +1,26 @@
+package model.response;
+
+public class BookAvailabilityResponse {
```

Comment:
Why should these response classes come under model? These do not inherit the basic properties of the model base.

Reply:
Since we are arranging data to be displayed to the customer, maybe it can come as part of views module?